app/assets/javascripts/initializers/initScrolling.js.erb

```diff
@@ -179,40 +179,46 @@ function fetchNextPodcastPage(el) {
   fetchNext(el, "/api/podcast_episodes", insertArticles)
 }
 
-function paginate(tag, requiresApproval) {
-  var tagFilters = [];
-  if (tag && tag.length > 0) {
-    tagFilters.push(tag);
-  }
+function paginate(tag, params, requiresApproval) {
+  const searchHash = { ...{ per_page: 15, page: nextPage }, ...JSON.parse(params) };
 
-  var searchObj = {
-      hitsPerPage: 15,
-      page: nextPage,
-      attributesToHighlight: [],
-      tagFilters,
-      filters: (requiresApproval === 'true') ? 'approved:true' : ''
+  if (tag && tag.length > 0) {
+    searchHash["tag_names"] = (searchHash["tag_names"] === undefined) ? [] : searchHash["tag_names"];
```

Comment:
If `searchHash["tag_names"]` is `null` or `undefined`, it will take the value of `[]`

```suggestion
    searchHash["tag_names"] = searchHash["tag_names"] || [];
```

Reply:
I totally thought I tried this in a console and it didnt work, I just retried it and it does. 🤦‍♀  Thank you!